### PR TITLE
Write Streamlink to event attachment instead of calendar by default

### LINKED ISSUE #7: Bugfix: Put Link in Event Attachments
Write the link to the livestream into the attachments section of the event, not in the link of the calendar entry to prevent errors with repeating events

### PR DESCRIPTION
**Pull request by JonFStr**
closes #7

### CODE REVIEW

**Comment by JonFStr** [comment]
closes #7

**Comment by alexhor** [comment]
I can't see the exact changes you made, as you overwrite entire files (my guess is it's because auf difference between windows and linux line endings).
Please resolve this, so I can take a closer look at your changes

File: Event.class.php [hunk 1]
```diff
@@ -261,22 +296,33 @@ public function deleteBroadcast() {
     $this->youtube->deleteBroadcast($this->broadcast);
     // Update the event
     unset($this->broadcast);
-    $this->link = new Link('');
+    $this->streamLink = null;
+    if ($this->livestreamInCalendar) {
```
Comment (alexhor) [inline Event.class.php:300]: Shouldn't this condition be negated?

Comment (alexhor) [reply Event.class.php:300]: Sorry, my bad. It shouldn't.

**Review by alexhor — APPROVED** [review]

**Pull request merged**